config/environments/test.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-
+  config.middleware.use RackSessionAccess::Middleware
```

Comment:
exactly, it's for .set_rack_session() in the feature tests to login a particular user